src/Authenticator/FormAuthenticator.php

```diff
@@ -115,12 +115,12 @@ public function authenticate(ServerRequestInterface $request, ResponseInterface
             ]);
         }
 
-        $user = $this->_identifier->identify($data);
+        $identity = $this->_identifier->identify($data);
```

Comment:
Actually this is still a *user* or *user/identity data*. It becomes an *identifier* (`IdentifierInterface` instance) later in authentication service.